packages/peregrine/lib/talons/Gallery/useGallery.js

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react';
+import { useQuery } from '@apollo/client';
+
+import { useUserContext } from '../../context/user';
+import mergeOperations from '../../util/shallowMerge';
+import defaultOperations from './gallery.gql';
```

Comment:
Not a huge deal and not sure if we wanted it as a standard but to be consistent with other talons can we use `DEFAULT_OPERATIONS` instead.

Reply:
Agreed on consistency, but I shifted here to being consistent with the other usage of `shallowMerge` as with `defaultClasses`. Let's align and I'll fix this up 👍

Reply:
Discussed this in peer review; largely this is personal preference, but for consistency we prefer to align on `defaultOperations`, sticking with the `defaultClasses` convention established in venia-ui. All imports are read-only so the constant syntax was a bit redundant, and probably introduced by me during the operation colocation re-factor. This syntax can be useful to indicate named export should be treated as immutable, but is less indicative on a default export.